project_generator/tools/tool.py

```diff
@@ -15,8 +15,9 @@
 import os
 import logging
 
-from os.path import join, dirname
-from jinja2 import Template
+from os.path import join, dirname, abspath
```

Comment:
There you go, now no one has the context to understand why you made this comment. :P